msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1040,7 +1040,7 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
 |`tabs.groups.controls.items.supertip`| | |✔️| Configures a supertip for the custom item. A supertip is a UI feature that displays a brief box of help information about a control when the cursor hovers over it. The box may contain multiple lines of text. |
 |`tabs.groups.controls.items.supertip.title`| String | 64 characters | ✔️ | Specifies the title text of the supertip.|
 |`tabs.groups.controls.items.supertip.description`| String | 128 characters | ✔️ | Specifies the description of the supertip.|
-|`tabs.groups.controls.items.actionId`| String | 64 characters | ✔️ | Specifies the ID of the action that is taken when a user selects the control or menu item. The `actionId` must match with `runtime.actions.id`. |
+|`tabs.groups.controls.items.actionId`| String | 64 characters | | Specifies the ID of the action that is taken when a user selects the control or menu item. The `actionId` must match with `runtime.actions.id`. |
```

Comment:
The property `actionId` in the  `extensionCommonCustomControlMenuItem` is marked as required in the [Dev Preview schema](https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json).

Is the update coming as part of a new schema version? Request you to please check and confirm.

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/3233253e-1746-4278-83e5-356b6acbfaa8)

Reply:
Confirmed with final schema update